.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Make sure you can reproduce with the latest version available**
+`pip install milatools --upgrade`
+
```

Comment:
I think a `What command did you run?` section would work well here, towards the beginning.

Reply:
Added